samigo/samigo-app/src/webapp/jsf/author/authorSettings.jsp

```diff
@@ -51,14 +51,15 @@
         $(document).ready(function() {
           // set up the accordion for settings
           var accordionPanel = 1;
-          if (window.sessionStorage && window.sessionStorage.getItem('samigo_assessmentsettings')) {
-              accordionPanel = parseInt(window.sessionStorage.getItem('samigo_assessmentsettings'));
+          var itemName = "samigo_assessmentsettings_" + <h:outputText value="#{assessmentSettings.assessmentId}"/>;
```

Comment:
Looks like you could just simplify this a bit as 
              var itemName = "samigo_assessmentsettings_${assessmentSettings.assessmentId}";

Same for published

Reply:
Looks good though!